includes/payments/class-give-sequential-donation-number.php

```diff
@@ -64,6 +66,7 @@ public function init() {
 	 * Set serialize donation number as donation title.
 	 * Note: only for internal use
 	 *
+     * @unreleased replace wp_update_post with DB::update
```

Comment:
```suggestion
     * @unreleased replace wp_update_post with DB::update to avoid affecting the post update date and invalidating the donation model's updatedAt date
```